.github/workflows/docker-buildx.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and publish docker image
+
+on:
+  workflow_run:
+    workflows: ["Release"]
+    types:
+      - completed
+
+jobs:
+  buildx_and_push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.workflow_run.head_branch }}
+
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+
+      - name: Get the version
+        id: get_version
+        run: |
+          version=$(git describe --tags --abbrev=0)
+          echo $version
+          echo ${version:1}
+          echo ::set-output name=VERSION::$version
+          echo ::set-output name=VERSION-NO-V::${version:1}
+        shell: bash
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and publish docker image
+        if: ${{ !contains(steps.get_version.outputs.VERSION, '-') }}
+        uses: docker/build-push-action@v2
+        with:
+          context: docker/release
+          file: ./docker/release/Dockerfile.release
+          build-args: RELEASE=${{ steps.get_version.outputs.VERSION-NO-V }}
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          tags: |
+            docker.io/alephium/explorer-backend:${{ steps.get_version.outputs.VERSION }}
+            docker.io/alephium/explorer-backend:latest
```

Comment:
There will be a race condition if this pushes at the same time with `docker.yml`.

Let's change this line in `build.sbt`:
```
-      val baseImageName = "alephium/explorer-backend"
+     val baseImageName = "alephium/dev-explorer-backend"
```